README.md

```diff
@@ -172,6 +172,13 @@ Options:
     --imappasswd passwd  IMAP account password
     --imapport port      Use a custom port
     --imapuser username  Who you login as
+    --imapproxyuser userbox     Use proxyauth on IMAP host. If set, allows
```

Comment:
I looked around, and the help text for "mbox" implies that this is a folder name (e.g. an "INBOX").

Here for proxyuser case, this is a user account name (e.g. the boss whose messages you manage while authenticated as a secretary's "username" with his "passwd", or you clean up a random email user while authenticated as a mail server admin).

Pending PR with "userbox" changed to "enduser" for clarity (hopefully).